src/verifications/verifications.ts

```diff
@@ -1404,6 +1436,58 @@ export class Verifications extends Logger {
     this.deleteFromVerificationCache('*', topic);
   }
 
+  /**
+   * Signs a transaction from an identity (offchain) and returns data, that can be used to submit it
+   * later on. Return value can be passed to ``executeTransaction``.
+   *
+   * Note that, when creating multiple signed transactions, the ``nonce`` argument **has to be
+   * specified and incremented between calls**, as the nonce is included in transaction data and
+   * restricts the order of transactions, that can be made.
+   *
+   * @param      {any}     contract             target contract of transcation or ``null`` if just sending funds 
+   * @param      {string}  functionName         function for transaction or ``null`` if just sending funds
+   * @param      {any}     options              options for transaction, supports from, to, nonce, input, value
+   * @param      {any[]}   args                 arguments for function transaction
+   * @returns    {VerificationsDelegationInfo}  prepared transaction for ``executeTransaction``
+   */
+  public async signTransaction(
+    contract: any = null,
+    functionName: string = null,
+    options: any,
```

Comment:
you could also already destructure the options here and set defaults if needed e.g.:
{value: = 0, nonce, input, .... }

Reply:
Yes, but I used this way of writing options this way to keep the semantic grouping of the web3 transaction argument `options` alive. `value` could be assigned with a simple default, the rest would clutter up the function head. As the properties of `options` are updated/replaced conditionally and given to other function calls as single variables, either the destructured or the passed variables had to be renamed, when not wishing to overwrite input variables like:

```
function example1({ to }, someCheckValue) {
  const toForOtherFunction = someCheckValue ? to : null;
  someOtherFunction(toForOtherFunction);
}
// or
function example2({ to: toArgument }, someCheckValue) {
  const to = someCheckValue ? toArgument : null;
  someOtherFunction(to);
}
```